big_tests/tests/gdpr_SUITE.erl

```diff
@@ -304,19 +322,25 @@ csv_to_maps(ExpectedHeader, [ExpectedHeader | Rows]) ->
 csv_row_to_map(Header, Row) ->
     maps:from_list(lists:zip(Header, Row)).
 
-validate_personal_maps(_, []) -> ok;
-validate_personal_maps([Map | RMaps], [Checks | RChecks]) ->
+validate_personal_maps(PersonalMaps, ExpectedItems) ->
```

Comment:
This change affects all other PRs, isn't it going to break all other testcases?

Reply:
actually its fixing them, as the order of incoming stanzas can result in bas assertion